src/Microsoft.Performance.SDK.Tests/DataTypes/TestRecord.cs

```diff
@@ -14,12 +14,12 @@ public class TestRecord
 
         public int CompareTo(int otherKey)
         {
-            return this.Key.CompareTo(otherKey);
+            return Key.CompareTo(otherKey);
         }
 
         public int GetKey()
         {
-            return this.Key;
```

Comment:
why this change? we usually keep this prefix on properties I thought

Reply:
not sure why this is here, I'll revert. thanks.